src/main/java/net/apnic/rdap/authority/RDAPAuthorityConfiguration.java

```diff
@@ -0,0 +1,128 @@
+package net.apnic.rdap.authority;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import javax.annotation.PostConstruct;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties
```

Comment:
{suggestion} Using a prefix will help keep this configuration compartmentalised.
ps, Q: Am I reading it right in that this class only add the 'authorities' key to the application properties config? (It also constructs the beans/etc, but I'm only asking about application.properties effects)